osu.Game.Tests/Visual/Playlists/TestScenePlaylistsResultsScreen.cs

```diff
@@ -42,15 +42,15 @@ public partial class TestScenePlaylistsResultsScreen : ScreenTestScene
         private int totalCount;
         private ScoreInfo userScore;
 
-        [SetUpSteps]
-        public override void SetUpSteps()
+        // We don't use SetUpSteps for this because one of the tests shouldn't receive any scores.
+        public void InitialiseUserScoresAndBeatmap(bool noScores = false)
         {
             base.SetUpSteps();
```

Comment:
I see some test failures like:
```
Failed test found in:
  TestResults-Windows-MultiThreaded.trx
Error:
  TearDown : osu.Framework.Testing.Drawables.Steps.AssertButton+TracedException : check first frame time: Expected: -10000.0d +/- 100.0d
    But was:  -9552.1072999999997d
    Off by:   -447.89270000000033d
```
Are these unrelated to this PR or does `base.SetUpSteps` perhaps still need to be in a `[SetUpSteps]` function to fix that?

Reply:
not related